packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -248,11 +286,31 @@
 	height: 20px;
 	align-items: center;
 	// stylelint-disable-next-line scales/font-size
-	font-size: 10px; //typography-exception smaller size needed
+	font-size: 10px; // typography-exception smaller size needed
 	text-transform: uppercase;
 	vertical-align: middle;
 	background-color: var( --studio-blue-50 );
 	color: var( --color-text-inverted );
+	margin: 2px 0; // harmless in desktop, useful in mobile (when the badge is below the text)
+}
+
+.domain-picker__suggestion-item.type-individual-item {
+	.domain-picker__badge {
+		color: var( --studio-green-80 );
+		background-color: var( --studio-green-5 );
+		// stylelint-disable-next-line scales/radii
+		border-radius: 4px;
+		text-transform: unset;
+		// stylelint-disable-next-line scales/font-size
+		font-size: 12px; // typography-exception smaller size needed
+	}
+
+	&.is-selected {
+		.domain-picker__badge {
+			color: $accent-blue;
+			background-color: #d9e8f7;
+		}
```

Comment:
The contrast ratio of the badge in the "selected" state is not accessible

![Screenshot 2020-11-05 at 19 55 55](https://user-images.githubusercontent.com/1083581/98284276-27a12500-1fa1-11eb-91af-2182b0928c6e.png)

Reply:
Thanks for checking this @ciampo! Can we please try the foreground color of the label inside the badge (shown in the screenshot) as #0F6CB3. I believe that's what it is in the designs, although I could have copy/pasted it incorrectly (which I'm known to do!)

Reply:
Thanks, Ollie! You're right, I made a mistake assuming the color is the same of the border. I changed to what you recommend and it's better but contrast was still under the recommended threshold. I cheated a bit and changed the background from `d9e8f7` to `ddebf8`.